configs/anyrtc.json

```diff
@@ -1,10 +1,72 @@
 {
   "index_name": "anyrtc",
   "start_urls": [
-    "https://docs.anyrtc.io/rtc-android/docs/"
+    {
+      "url": "https://docs.anyrtc.io/",
+      "tags": ["all"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-web/",
+      "tags": ["rtc-web"],
+      "variables": {
```

Comment:
Same feedback as https://github.com/algolia/docsearch-configs/pull/2285/files#r467761459
This review apply to all items

Reply:
This point is not resolved. @chudongvip Can you clarify why you marked this as resolved?